tests/ert3/config/test_stages_config.py

```diff
@@ -48,46 +63,21 @@ def test_entry_point_not_valid(config, expected_error):
         ert3.config.load_stages_config(config)
 
 
-def test_step_valid():
-    config = ert3.config.load_stages_config(
-        [
-            {
-                "name": "some_name",
-                "type": "unix",
-                "input": [{"record": "some_record", "location": "some_location"}],
-                "output": [{"record": "some_record", "location": "some_location"}],
-                "transportable_commands": [{"name": "poly", "location": "poly.py"}],
-                "script": ["poly --help"],
-            }
-        ]
-    )
-    assert config[0].name == "some_name"
+def test_step_valid(base_unix_stage_config):
+    config = ert3.config.load_stages_config(base_unix_stage_config)
+    assert config[0].name == "unix_stage"
```

Comment:
This test seems to be the same as `test_entry_point`, with one extra assert added. Are both tests needed?

Reply:
I was also confused about both tests were required, I did not want to remove one of them when I noticed this issue, but I agree one of them should be removed.